osu.Game/Screens/Loader.cs

```diff
@@ -58,9 +59,16 @@ protected virtual OsuScreen CreateLoadableScreen()
         }
 
         private IntroScreen getIntroSequence()
-        {
+        {//вот именно что не показывает ни всплывающих подсказок нихера
```

Comment:
Please keep all source, including comments, in English. Additionally your whitespace needs fixing (as reported by CI).